documentation/tarron_notes.md

```diff
@@ -2,12 +2,33 @@
 
 ## Questions
 
---
+- [ ] In the following code, it looks like each badge will have a "user" property that will return a user. Since we decided to just have one badge and we'll tie that to all the users that have earned it, should we get rid of this?
```

Comment:
Maybe instead of returning a single user, it returns a list of users? That would be possible with the current data structure, and would be valuable information to have.